packages/dds/merge-tree/src/collections.ts

```diff
@@ -390,18 +396,14 @@ export interface RBNodeActions<TKey, TData> {
     showStructure?: boolean;
 }
 
-export class RedBlackTree<TKey, TData> implements Base.SortedDictionary<TKey, TData> {
+export class RedBlackTree<TKey, TData> implements SortedDictionary<TKey, TData> {
     root: RBNode<TKey, TData> | undefined;
-    constructor(public compareKeys: Base.KeyComparer<TKey>, public aug?: IRBAugmentation<TKey, TData>) {
+    constructor(public compareKeys: KeyComparer<TKey>, public aug?: IRBAugmentation<TKey, TData>) {
 
     }
 
     makeNode(key: TKey, data: TData, color: RBColor, size: number) {
-        const node = <RBNode<TKey, TData>>{ key, data, color, size };
-        if (this.aug && this.aug.init) {
-            this.aug.init(node);
```

Comment:
This is the one functional change - I was looking to type the `init` method but found no instances to base the typing on.  I elected to remove it since I don't believe we have direct external customers of `RedBlackTree`.